src/components/App/Post/PostQuickAdd.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { uniqueId } from 'lodash';
+
+import { Input } from '../../Form';
+
+const generatePostUrl = () => {
+  const imgNumber = Math.floor(Math.random() * 100);
+  return `https://picsum.photos/320/240?image=${imgNumber}`;
+};
+
+class PostQuickAdd extends Component {
+  emptyPost = {
+    title: '',
+    body: '',
+  };
+
+  state = {
+    ...this.emptyPost,
+  };
+
+  handleChange = ({ target: { value, name } }) => {
+    this.setState({ [name]: value });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    this.props.onSubmit({
+      ...this.state,
+      id: uniqueId('new'),
+      imageUrl: generatePostUrl(),
+    });
+
+    this.setState(this.emptyPost);
```

Comment:
empty post doesn't have to be class property. It can be variable outside of react component scope